release.yaml

```diff
@@ -65,4 +65,4 @@ dependencies:
     - addon: 'script.module.addon.signals'
       version: '0.0.5+matrix.1'
     - addon: 'script.module.websocket'
-      version: '0.58.0+matrix.1'
+      version: '1.6.4'
```

Comment:
Have you verified that this version of the websocket library actually exists in the Kodi repositories for the supported versions?

Reply:
Yes this version is available for Matrix, Nexus and Omega